modules/custom/cu_inactive_users/cu_inactive_users.module

```diff
@@ -375,3 +388,96 @@ function _cu_inactive_users_format_roles($roles) {
   $roles = ucwords($roles);
   return $roles;
 }
+
+function cu_inactive_users_report_remove($uids = NULL) {
+  cu_inactive_users_delete_dev_accounts();
+  $output = array();
+  $list = array();
+  $results = cu_inactive_users_report_remove_process($uids);
+  foreach ($results as $result) {
+    $name = !empty($result->realname) ? l($result->realname, 'user/' . $result->uid . '/edit') : l($result->name, 'user/' . $result->uid . '/edit');
+    $key = strip_tags($name);
+    $list[$key] = $name;
+  }
+  ksort($list);
+  $output['missing_role']['#prefix'] = '<h2>Can Probably Be Removed</h2>';
+  $output['missing_role']['description']['#markup'] = '<p>The users listed below have no roles assigned to them, have not authored or edited any content, and have not logged in recently.</p>';
+  $output['missing_role']['list']['#markup'] = theme('item_list', array('items' => $list, 'attributes' => array('class' => array('bullet-list'))));
+
+  return $output;
+}
+
+function cu_inactive_users_report_remove_process($uids) {
+  $time = 60*60*24*395;
+  $lists = array();
+  $lists['stale'] = cu_inactive_users_stale_logins($time, $uids);
+  $lists['role'] = cu_inactive_users_missing_role_get($uids);
+  $lists['edits'] = cu_inactive_users_noedits($uids);
+  //$results = array_intersect_key($list['stale'], $list['role'], $list['edits']);
+
+  $results = array();
+  foreach ($lists as $list) {
+    if (!empty($list)) {
+      if (empty($results)) {
+        $results = $list;
+      }
+      else {
+        $results = array_intersect_key($results, $list);
+      }
+    }
+  }
+  return $results;
+}
+
+function cu_inactive_users_delete_dev_accounts() {
```

Comment:
could be https://github.com/CuBoulder/express/pull/975/files#diff-48408eeac819afb811baba92268b7c93R347 since "all" gets all of the users.